client/src/commands/roll.ts

```diff
@@ -0,0 +1,60 @@
+import { LOG_EVENT, ROLL_EVENT } from "../../../events"
+import { LogItem } from "../components/terminal"
+import { sendEvent } from "../network"
+import { commandEmitter, CommandModule } from "./emitter"
+
+export const diceRegex =/(\d+)d(\d+)/;
+
+export type DiceProps = {
+  count: number;
+  sides: number;
+}
+
+const defaultRoll: DiceProps = {
+  count: 1,
+  sides: 69,
+}
+
+const parseDice = (value: string) => {
+  const match = value.match(diceRegex);
+  if (!match) return false;
+  if (match.length < 3) return false;
+  return {count: parseInt(match[1]), sides: parseInt(match[2])};
```

Comment:
this should be split onto multiple lines to avoid conflicts

Reply:
This makes so much sense and I've never thought about it